onnx/defs/tensor/defs.cc

```diff
@@ -15,22 +15,47 @@ in the 'DataType' enum field in the TensorProto message.
 NOTE: Casting to and from strings is not supported yet.
 )DOC")
     .Attr(
-          "to",
-          "The data type to which the elements of the input tensor are cast."
-          "Strictly must be one of the types from DataType enum in TensorProto",
-          AttributeProto::STRING)
+        "to",
+        "The data type to which the elements of the input tensor are cast."
+        "Strictly must be one of the types from DataType enum in TensorProto",
+        AttributeProto::STRING)
     .Input(0, "input", "Input tensor to be cast.", "T1")
     .Output(
         0,
         "output",
         "Output tensor with the same shape as input with type "
         "specified by the 'to' argument",
         "T2")
-    .TypeConstraint("T1", { "tensor(float16)", "tensor(float)", "tensor(double)" },
-            "Constrain input types to float tensors.")
-    .TypeConstraint("T2", { "tensor(float16)", "tensor(float)", "tensor(double)" },
-            "Constrain output types to float tensors.");
-
+    .TypeConstraint(
```

Comment:
This is the only change in this file, all the other changes are due to format changes after running clang-format against this file, and I kept the format changes as they're. Thanks!